images/sd_charging_profile_updated_via_msp.svg

```diff
@@ -1 +1 @@
(#f117k9pquvqadi)" height="58.2656" style="stroke: #A80036; stroke-width: 1.0;" width="10" x="230.5" y="124.8594"/><rect fill="#FFFFFF" filter="url(#f117k9pquvqadi)" height="72.2656" style="stroke: #A80036; stroke-width: 1.0;" width="10" x="615.5" y="169.125"/><rect fill="#FFFFFF" filter="url(#f117k9pquvqadi)" height="14" style="stroke: #A80036; stroke-width: 1.0;" width="10" x="1000.5" y="227.3906"/><path d="M20,69.5938 L20,94.5938 L1046,94.5938 L1046,79.5938 L1036,69.5938 L20,69.5938 " fill="#FBFB77" filter="url(#f117k9pquvqadi)" style="stroke: #A80036; stroke-width: 1.0;"/><path d="M1036,69.5938 L1036,79.5938 L1046,79.5938 L1036,69.5938 " fill="#FBFB77" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="178" x="437.75" y="86.6606">Ongoing Session with id=15</text><polygon fill="#A80036" points="218.5,120.8594,228.5,124.8594,218.5,128.8594,222.5,124.8594" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="61.5" x2="224.5" y1="124.8594" y2="124.8594"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="145" x="68.5" y="119.7935">update ChargingProfile</text><polygon fill="#A80036" points="603.5,165.125,613.5,169.125,603.5,173.125,607.5,169.125" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="240.5" x2="609.5" y1="169.125" y2="169.125"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="356" x="247.5" y="148.9263">PUT https://www.server.com/ocpi/2.2/chargingprofiles/15</text><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="145" x="279.5" y="164.0591">ActiveChargingProfile()</text><polygon fill="#A80036" points="246.5,179.125,236.5,183.125,246.5,187.125,242.5,183.125" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 2.0,2.0;" x1="240.5" x2="614.5" y1="183.125" y2="183.125"/><polygon fill="#A80036" points="988.5,223.3906,998.5,227.3906,988.5,231.3906,992.5,227.3906" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="625.5" x2="994.5" y1="227.3906" y2="227.3906"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="356" x="632.5" y="207.1919">PUT https://www.server.com/ocpi/2.2/chargingprofiles/15</text><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="145" x="664.5" y="222.3247">ActiveChargingProfile()</text><polygon fill="#A80036" points="631.5,237.3906,621.5,241.3906,631.5,245.3906,627.5,241.3906" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 2.0,2.0;" x1="625.5" x2="1004.5" y1="241.3906" y2="241.3906"/></g></svg>
```

Comment:
Please change the PlantUML file, that is the source from which the diagram is created. This SVG file will be overwritten during the generation of the PDF.

Reply:
Done.